sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_helpers.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from datetime import time
+from azure.core.exceptions import HttpResponseError
+
+POLLING_INTERVAL = 1
+
+
+def get_pipeline_response(pipeline_response, _, response_headers):  # pylint: disable=unused-argument
+    return pipeline_response
+
+
+def get_field_scalar_value(field):  # pylint: disable=too-many-return-statements
+    field_type = field.type
+    if field_type == "string":
+        return field.value_string
+    if field_type == "number":
+        return field.value_number
+    if field_type == "integer":
+        return field.value_integer
+    if field_type == "date":
+        return field.value_date
+    if field_type == "phoneNumber":
+        return field.value_phone_number
+    if field_type == "time":
+        hour, minutes, seconds = field.value_time.split(":")
+        return time(int(hour), int(minutes), int(seconds))
+    return None
+
+
+def get_content_type(form):
+    """Source: https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_numbers_in_files
+    """
+    if len(form) > 3:
+        if form[0] == 0x25 and form[1] == 0x50 and form[2] == 0x44 and form[3] == 0x46:
```

Comment:
I would make something more like
```python
if form[:4] == b'%PDF'
```
if really you want to hexidecimal form:
```python
if form[:4] == bytes([0x25, 0x050, 0x44, 0x046])
```

Reply:
Fixed